tools/bootstrapAdmin.js

```diff
@@ -0,0 +1,28 @@
+import { User, Permission, sequelize } from '../src/models';
+import logger from '../src/logger';
+import SCOPES from '../src/middleware/scopeConstants';
+
+const { ADMIN } = SCOPES;
+
+export const ADMIN_EMAIL = "ryan.ahearn@gsa.gov";
+
+const bootstrapAdmin = async () => {
+  const user = await User.findOne({ where: { email: ADMIN_EMAIL } });
+  if (user === null) {
+    throw new Error(`User ${ADMIN_EMAIL} could not be found to bootstrap admin`);
+  }
+
+  const [permission, created] = await Permission.findOrCreate({
+    where: {
+      userId: user.id,
+      scopeId: ADMIN,
+      regionId: 14
```

Comment:
👍 We should be ok if this is limited to region 14.

Reply:
I thought `14` was meant for "non-regional" things. Should this actually be something else?

Reply:
Yeah `14` is for "non-regional" permissions

Reply:
> Yeah `14` is for "non-regional" permissions

Which doesn't feel that great... maybe we should allow the region to be null?

Reply:
> Which doesn't feel that great (wish I had come up with something better!)... maybe we should allow the region to be null?

I think this is something to revisit as we work on more central office-focused stories.